assets/app/view/create_game.rb

```diff
@@ -545,5 +520,146 @@ def update_inputs(title_change: false)
       store(:optional_rules, @optional_rules, skip: true)
       store(:visible_optional_rules, visible_rules)
     end
+
+    def select_game(meta)
+      @selected_game = meta
+
+      update_player_range(meta)
+
+      uncheck_game_variant
+      @selected_variant = nil
+      @game_variants = {}
+
+      uncheck_optional_rules
+      @optional_rules = []
+
+      preselect_variant(@selected_game)
+      preselect_optional_rules(@selected_game)
+
+      update_inputs(title_change: true)
+
+      `window.scroll(0, 0)`
```

Comment:
re: https://github.com/tobymao/18xx/pull/7923#issuecomment-1331657847

> FWIW I think the game buttons are kind of ugly. What if the title wasn't a button and instead when you click on a row, it is highlighted somehow? That's what the spreadsheet and info tab do, so it's a little more on brand.

I like this suggestion, I'll try that out.

> Also, when a game is selected (say way down in the list), what happens? Does the page move to the top, so the user can enter the game details and hit the create button?

Yep, here at the end of `select_game()` the user is sent to the top of the page

Reply:
Here's how it looks with the button removed and the row highlighting the active game:

![Screenshot 2022-12-03 202637](https://user-images.githubusercontent.com/1045173/205472881-d7e49155-28a6-40de-89dd-b8913e781870.png)

Reply:
With this setup the whole row is clickable, but it doesn't really look like it unless you're on a computer and can see the cursor change on hover. 🤷

Reply:
@crericha I'll go ahead and merge like this, we can make further UI tweaks later if necessary